src/test/scala/TypeCheckerSpec.scala

```diff
@@ -1278,6 +1278,16 @@ class TypeCheckerSpec extends FunSpec {
         let z: double = x + y;
         """ )
     }
+
+    it("unsigned and signed cannot be compared") {
```

Comment:
```suggestion
    it("unsigned and signed types are incomparable") {
```

(using the magic word "incomparable," which literally means the same thing but evokes partial ordering)